lucene/ivy-versions.properties

```diff
@@ -203,6 +203,8 @@ org.carrot2.morfologik.version = 2.1.1
 /org.carrot2/morfologik-polish = ${org.carrot2.morfologik.version}
 /org.carrot2/morfologik-stemming = ${org.carrot2.morfologik.version}
 
+ua.net.nlp.morfologik-ukrainian-search.version = 3.7.4
```

Comment:
This is fine, but you're missing the `/org/name=version` thing that's required to be used in `ivy.xml` files.

If you look at the rest of this file, you can see that some `/org/name` keys share the same version, and so a variable is introduced to simplify maintenance.  In your case, when there is only one `/org/name` key using this version, you should IMO eliminate the `ua.net.nlp.morfologik-ukrainian-search.version` variable entirely and add `/ua.net.nlp/morfologik-ukrainian-search = 3.7.4`.